pages/index.js

```diff
@@ -1,34 +1,79 @@
 import Head from "next/head";
 import styled from "styled-components";
-import { Inter } from "next/font/google";
-import EntryForm from "@/components/EntryForm/EntryForm";
+import { useState, useEffect } from "react";
+import { useRouter } from "next/router";
+import { CircularProgressbar, buildStyles } from "react-circular-progressbar";
+import 'react-circular-progressbar/dist/styles.css';
+import Heading from "@/components/Header/Header";
 
-const inter = Inter({ subsets: ["latin"] });
+export default function HomePage({ entries }) {
+  const [co2Emission, setCo2Emission] = useState(0);
+  const router = useRouter();
+
+  useEffect(() => {
+    const currentDate = new Date();
+    const currentMonth = currentDate.getMonth();
+
+    const monthlyEmissions = entries.reduce((total, entry) => {
+      const entryDate = new Date(entry.date);
+      const entryMonth = entryDate.getMonth();
+
+      if (entryMonth === currentMonth) {
+        total += entry.result;
+      }
+
+      return total;
+    }, 0);
+
+    setCo2Emission(monthlyEmissions);
+  }, [entries]);
 
-export default function Home({ onSubmit, result, onCalculateCo2 }) {
   return (
     <>
       <Head>
-        <title>Capstone Project</title>
-        <meta name="description" content="Penguin Capstone Project" />
+        <title>
+          CO<sub>2</sub>Mobility Tracker
+        </title>
+        <meta name="description" content="Dona Capstone Project" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
-      <main className={inter.className}>
+      <main>
         <Heading>
-          CO<sub>2</sub>-Mobility-Tracker
+          CO<sub>2</sub> Mobility Tracker
         </Heading>
-        <EntryForm
-          formName="calculator"
-          onSubmit={onSubmit}
-          result={result}
-          onCalculateCo2={onCalculateCo2}
-        />
+        <StyleDiv>
+          <p>Hi Dona! This is what you have emitted this month:</p>
+        </StyleDiv>
+        <StyledBar>
+          <CircularProgressbar
+            value={co2Emission}
+            maxValue={2000}
+            text={`${co2Emission.toFixed(2)} kg`}
+            styles={buildStyles({
+              pathColor: "#3498db",
+              textColor: "#3498db",
+              strokeLinecap: "round",
+              pathTransitionDuration: 1.5,
+              trailColor: "#d6d6d6",
+              textSize: "12px",
+            })}
+          />
+        </StyledBar>
+        <button type="button" onClick={() => router.push("/calculator/")}>
+          Let's add more journeys!
```

Comment:
In case you haven't yet found the source of the error on Vercel, here it is. You need to escape the apostrophe character or use a different one.

Reply:
Hi Ernst, yes I had found it. I also remembered that Alexandra had the same issue in the past. Now I have fixed it.